src/java.management/share/classes/javax/management/modelmbean/RequiredModelMBean.java

```diff
@@ -1192,10 +1190,6 @@ public Object run() {
                 }
             }
             return rslt;
-        } catch (RuntimeErrorException ree) {
-            throw new RuntimeOperationsException(ree,
-                      "RuntimeException occurred in RequiredModelMBean "+
-                      "while trying to invoke operation " + opName);
```

Comment:
This one is a bit different. You've collapsed RuntimeOperationsException into the RuntimeException handler, which seems fine since it is a subclass of RuntimeException and both handlers do the same thing. However, I wonder if the original intent of the RuntimeOperationsException handler was for it to have a different message, and perhaps reference RuntimeOperationsException instead of RuntimeException.

Reply:
The proposed change is behaviour preserving so I have no issue with it. If something else was intended then that would be a new RFE IMO.